components/messenger/transports.rst

```diff
@@ -0,0 +1,225 @@
+.. index::
+    single: Messenger; Transports
+
+Transports
+==========
+
+To send and receive messages via message brokers, the Messenger component has
+"transports". Transports are responsible for routing messages to and from
+the message brokers.
+
+Every transport is configurable using a DSN. This DSN allows you to chose the
+transport layer as well as configure it.
+
+AMQP
+----
+
+The most famous message broker protocol is probably AMQP, which is most
+commonly implemented with RabbitMQ. The Messenger component has built-in
+support for AMQP.
+
+How it works?
+~~~~~~~~~~~~~
+
+A DSN protocol that uses ``amqp`` is recognized and sent to the built-in AMQP transport.
+For example::
+
+    $dsn = 'amqp://user:password@localhost/%2f/messages'
+
+The messages will be sent to the ``messages`` exchange bound to the ``messages``
+queue on the ``/`` vhost (the ``%2f`` is a url-encoded ``/``).
+
+.. note:
+
+    By default, RabbitMQ uses ``guest`` as a username and ``guest`` as a password
+    and has a ``/`` vhost.
+
+Error handling
```

Comment:
This should be "Error Handling" (we capitialize all non-closed class words in titles)

Reply:
And is this special behavior for AMQP, or is it general behavior of the Message component? (I'm asking because the title underline makes it part of the "AMQP" section)